pyoracc/test/atf/test_atfserializer.py

```diff
@@ -113,23 +113,21 @@ def test_text_code_and_description():
     """
     Check if serializing works for the code/description case - first line
     of ATF texts.
-    Note the parser always returns an AtfFile object, even when it's not
-    ATF-compliant.
```

Comment:
Is this no longer true (from previous updates), or did you just not consider it relevant? I'm guessing that it was here to make it clear that the test is meaningful, as the serializer always works on an `AtfFile` object rather than a plain string (although with the more transparent method calls, maybe this doesn't matter).

Reply:
I guessed similarly. I removed it because don't think it's correct for the current code. In the case of the project line test:
```python
>>> from pyoracc.atf.common.atffile import AtfFile
>>> AtfFile("#project: cams/gkab\n")
[...]
SyntaxError: PyOracc could not parse token PROJECT at line 1 at offset 1 with value 'project'.
```
i.e. it fails without the preceding code and description line. I suppose it's partially correct in that if the required lines occur in order it will return an object with the missing values filled in with defaults, as happens with the code test, but either way I didn't find it helpful enough to include in the docstring.

These tests all look pretty easy to fix up, but I thought that should be a different PR.

Reply:
Makes sense, thanks for catching that then!